selenium/Kurs2021.cs

```diff
@@ -38,17 +58,83 @@ public void AddFields()
             driver = new ChromeDriver();
             driver.Navigate().GoToUrl("https://www.seleniumeasy.com/test/basic-first-form-demo.html");
             Thread.Sleep(2000);
+
+            var xOnPopup = driver.FindElement(By.Id("at-cv-lightbox-close"));
             driver.FindElement(By.Id("at-cv-lightbox-close")).Click();
             driver.FindElement(By.Id("sum1")).SendKeys("2");
             driver.FindElement(By.Id("sum2")).SendKeys("3");
             driver.FindElements(By.ClassName("btn-default"))[1].Click();
 
+
+
             var text = driver.FindElement(By.Id("displayvalue")).Text;
 
             Assert.AreEqual("5", text);
 
             driver.Quit();
+        }
+
+        [Test]
+
+        public void CheckCheckbox()
+        {
+            //arrange
+            driver = new ChromeDriver();
```

Comment:
You already have assigned new ChromeDriver in SetUp method. You don't need to do that again. In all tests. Please remove it.